boards/nucleo_f429zi/Makefile

```diff
@@ -8,18 +8,13 @@ include ../Makefile.common
 OPENOCD=openocd
 OPENOCD_OPTIONS=-f openocd.cfg
 
-# OpenOCD requires fullpath
```

Comment:
Did something change in OpenOCD that this is no longer true?

Reply:
The target is now specified with the full path, openocd still gets the fill path, but directly from $<.

Reply:
Ah, that makes sense -- thanks!